packages/amplify-provider-awscloudformation/src/aws-utils/aws-cognito-identity.js

```diff
@@ -0,0 +1,20 @@
+const aws = require('./aws.js');
+const configurationManager = require('../configuration-manager');
+
+class CognitoIdentity {
+  constructor(context, options = {}) {
+    return (async () => {
```

Comment:
Can we do this as an singleton with getInstance method

Reply:
Removed as it was not used in the end, also removed Cognito which was used by some method that was not referenced.